pages/product-zoom.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="zoom">
+    <div class="zoom__title">
+      <SfBar class="sf-modal__bar" :close="true" :title="productName" @click:close="closeModal" />
+    </div>
+    <div class="zoom__gallery">
+      <KiboGallery
```

Comment:
remove dev zoom page

Reply:
As per UX mockup we should display top header even after zooming the image,  hence I have created a separate page(zoom-product) for the same. Please let me know if you want me to use the Model popup instead of a separate page?